deployments/kubernetes/templates/chart/values.yaml.tmpl

```diff
@@ -6,6 +6,7 @@ kubernetes:
 ingressMonitorController:
   tolerations: {}
   watchNamespace: ""
+  mount: "secret"
```

Comment:
You made "secret" a default type. Shouldn't it be configmap?

Reply:
yes. changed. kindly review